icu4j/main/tests/core/src/com/ibm/icu/dev/test/timezone/TimeZoneRuleTest.java

```diff
@@ -1469,7 +1470,7 @@ public void TestBasicTimeZoneCoverage() {
             BasicTimeZone btz = (BasicTimeZone)tz;
             int []offsets = new int[2];
 
-            btz.getOffsetFromLocal(Calendar.getInstance().getTimeInMillis(), BasicTimeZone.LOCAL_STD, BasicTimeZone.LOCAL_STD, offsets);
+            btz.getOffsetFromLocal(Calendar.getInstance().getTimeInMillis(), LocalOption.STANDARD_FORMER, LocalOption.STANDARD_LATTER, offsets);
             if (offsets[0] > offsets[1]) {
```

Comment:
could we extend the test to cover the testing of all 6 possible values LocalOption for BasicTimeZone here?

Reply:
Same as above.

Reply:
could you file a ticket and add a TODO with that ticket number here at least. thanks